source/stun_deserializer.c

```diff
@@ -205,6 +290,13 @@ StunResult_t StunDeserializer_GetNextAttribute( StunContext_t * pCtx,
         {
             result = STUN_RESULT_OUT_OF_MEMORY;
         }
+
+        else if( pAttribute->attributeValueLength != isValidAttribute( pAttribute->attributeType,
+                                                                       pAttribute->attributeValueLength ) )
+        {
+            /* Check for malformed packets with wrong attribute length */
+            result = STUN_RESULT_MALFORMED_MESSAGE;
```

Comment:
Can we give it a more specific error name instead of malformed message, like `INVALID_ATTRIBUTE`? That might help users for debugging when it's happening.

Reply:
yes, sure I can change it.